### ISSUE TITLE
Handle <answer> appropriately

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/precalculus#110]
Right now, `<answer>`s are being displayed in the web, print, and slides versions. We should decide on, and implement, the desired behavior for this tag.

**Comment by siwelwerd** [comment]
Thought I'd take a quick stab at this this morning.  I think the 'right' way to do this is to add the following in our main publication file (since activities are project-likes):

`<exercise-project statement="yes" hint="yes" answer="no" solution="no" />`

Then, duplicate the publication file for an instructor.ptx file, and edit that line appropriately, and also update project.ptx with new targets.

However, I discovered something interesting: When you set `@answer="no"`, this will omit answers from nicely structured things, e.g.

```
<task>
<statement>
<p>Do something</p>
</statement>
<answer>
<p>The answer is <m>42</m>.</p>
</answer>
</task>.
```

On the other hand, PreTeXt will happily handle something like this as well:
```
<task>
<p>Do something</p>
<answer>
<p>The answer is <m>42</m>.</p>
</answer>
</task>.
```
But, in this case, even with `@answer` set to "no", it will produce the answers. 

All this to say, my question for @StevenClontz is: Is this indeed the 'right' way?  If so, we need to instruct our authors to ensure that the content of a `<task>` must be in a `<statement>`.  We talked about this last week and I indicated it would be better from a structural standpoint but ultimately probably didn't really matter--I didn't realize at the time PreTeXt would actually do something different with it.

**Comment by StevenClontz** [comment]
If a `<task/>` has any structure besides just pargraphs, it needs a `<statement/>`. So this is how we should structure `<task/>`s with `<answer/>`s:

```xml
<task>
  <statement>
    <p>Do something</p>
  </statement>
  <answer>
    <p>The answer is <m>42</m>.</p>
  </answer>
</task>
```

**Comment by siwelwerd** [comment]
So if you forget a `<p>` tag, PreTeXt just quietly omits things with no warning or error, and if you forget a `<statement>`, it just quietly includes it with no warning or error. Got it 🙃.

**Comment by StevenClontz** [comment]
Don't shoot the messenger. 😅

**Comment by siwelwerd** [comment]
Well, we've got some cleaning up to do... 😱

```
(base) drew@Drews-MacBook-Pro precalculus % xpath -q -e '//task[not(statement)]' source/*/*.ptx | grep "<task>" | wc -l
334
```

**Comment by siwelwerd** [comment]
Opened #133 to contain any further discussion of `<task>`s needing `<statement>`s.  That issue is currently blocking this one.

**Comment by siwelwerd** [comment]
This was closed by #132
